packages/engine/Source/Scene/Cesium3DTileset.js

```diff
@@ -68,7 +68,9 @@ import Cesium3DTilesetSkipTraversal from "./Cesium3DTilesetSkipTraversal.js";
  * @property {Axis} [modelForwardAxis=Axis.X] Which axis is considered forward when loading models for tile contents.
  * @property {ShadowMode} [shadows=ShadowMode.ENABLED] Determines whether the tileset casts or receives shadows from light sources.
  * @property {number} [maximumScreenSpaceError=16] The maximum screen space error used to drive level of detail refinement.
- * @property {number} [maximumMemoryUsage=512] The maximum amount of memory in MB that can be used by the tileset.
+ * @property {number} [maximumMemoryUsage=512] The maximum amount of memory in MB that can be used by the tileset. Deprecated.
+ * @property {number} [cacheBytes=536870912] The size (in bytes) to which the tile cache will be trimmed, if the cache contains tiles not needed for the current view.
+ * @property {number} [maximumCacheOverflowBytes=536870912] The maximum additional memory (in bytes) to allow for cache headroom, if more than {@link Cesium3DTileset#cacheBytes} are needed for the current view. Must be at least 10 * 1024 * 1024.
```

Comment:
> Must be at least 10 * 1024 * 1024

This seems a bit arbitrary. Why can't it be 0?

Reply:
We currently allow `maximumMemoryUsage == 0`, to force the cache to be trimmed to the minimum size possible. This is used several times in the specs, so I also allowed `cacheBytes == 0` for consistency.

If both `cacheBytes == 0` and `maximumCacheOverflowBytes == 0`, then no tiles will be processed and `memoryAdjustedScreenSpaceError` will grow to infinity. This feels buggy, as Matt noticed already, even with a small non-zero value.

I chose 10 MB as a number that should guarantee we render at least one tile.

Reply:
As discussed with @ggetz, I changed this to allow `maximumCacheOverflowBytes == 0`, and simply added a `oneTimeWarning` inside `increaseScreenSpaceError`, to alert the developer that the values of `cacheBytes` and `maximumCacheOverflowBytes` might be too small.